src_Core/Core/CoreW.bsv

```diff
@@ -215,46 +254,61 @@ module mkCoreW #(Reset dm_power_on_reset)
    TV_Encode_IFC tv_encode <- mkTV_Encode;
 `endif
 
+   function do_release (restartRunning) = action
+      plic.set_addr_map (zeroExtend (soc_map.m_plic_addr_range.base),
+                         zeroExtend (rangeTop(soc_map.m_plic_addr_range)));
+      proc.start (restartRunning, soc_map_struct.pc_reset_value, 0, 0);
+      //proc.set_verbosity (verbosity);
```

Comment:
Remove if not likely needed in the future.

Reply:
can do :) thx